### PR TITLE
Support TF 1.15 and 2.0 in tests

### PR DESCRIPTION
**Pull request by jantonguirao**
Signed-off-by: Joaquin Anton <janton@nvidia.com>

#### Why we need this PR?
Need to support latest TF versions

#### What happened in this PR?
Add TF 1.15 and 2.0 to the list of supported version for CUDA 10

**JIRA TASK**: [DALI-XXXX]

### CODE REVIEW

File: dali_tf_plugin/dali_tf_plugin_utils.py [hunk 1]
```diff
@@ -49,7 +49,14 @@ def get_tf_compiler_version():
...
     s = str(subprocess.check_output(cmd, shell=True))
-    version = re.search("GCC:\s*\(.*\)\s*(\d+.\d+).\d+", s).group(1)
+    lines = s.split('\\n')
+    version = ''
+    for line in lines:
+        res = re.search("GCC:\s*\(.*\)\s*(\d+.\d+).\d+", line)
+        if res:
+            ver = res.group(1)
+            if not version or version < ver:
+                version = ver
```
Comment (awolant) [inline dali_tf_plugin/dali_tf_plugin_utils.py]: I think that `group` returns `string` or `tuple`. In that case `<` is alphabetical order, I think. Maybe explicit `int(ver)` would be better here?

File: qa/TL1_tensorflow_plugin/test.sh [hunk 2]
```diff
@@ -6,7 +6,7 @@ target_dir=./dali/test/python
 do_once() {
     USE_CUDA_VERSION=$(cat /usr/local/cuda/version.txt | sed 's/.*Version \([0-9]\+\)\.\([0-9]\+\).*/\1/')
     if [[ "$USE_CUDA_VERSION" = "10" ]]; then
-        export TENSORFLOW_VERSIONS="1.13.1 1.14"
+        export TENSORFLOW_VERSIONS="1.13.1 1.14 1.15 2.0"
     else
         export TENSORFLOW_VERSIONS="1.7 1.8 1.9 1.10 1.11 1.12"
```
Comment (JanuszL) [inline qa/TL1_tensorflow_plugin/test.sh]: I think in the follow up we should deprecate some 0.9 versions (and maybe 1.13.1 as well)

File: dali_tf_plugin/dali_tf_plugin_utils.py (hunk 1 above)
Comment (jantonguirao) [reply dali_tf_plugin/dali_tf_plugin_utils.py]: done

File: qa/TL1_tensorflow_plugin/test.sh (hunk 2 above)
Comment (awolant) [inline qa/TL1_tensorflow_plugin/test.sh]: I think these lines require synchronization with `setup_packages.py`

**Pull request merged**